website/content/en/docs/main/tutorials/nginx/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Nginx
+description: Running basic connectivity between Nginx server and client across two clusters using ClusterLink.
+---
+
+In this tutorial, we'll establish connectivity across clusters using ClusterLink to access a remote Nginx server.
+The tutorial uses two kind clusters:
```

Comment:
```suggestion
The tutorial uses two clusters:
```
Since in the initialize clusters you say that one can also use existing clusters...